.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: java
+sudo: false
+
+jdk:
+  - oraclejdk8
+  - oraclejdk9
+  - oraclejdk11
+  - openjdk8
+  - openjdk9
+  - openjdk10
+  - openjdk11
+  - openjdk12
```

Comment:
Are all these JDKs really required? Shouldn't we focus only on the most important (LTS) ones - those would be 8, 11  and maybe latest available non-LTS (12)?

Keep in mind that in the end, number of executors available to all projects in eclipse-ee4j org is limited.

Reply:
reasonable. what about:
```
  - oraclejdk8
  - oraclejdk11
  - openjdk8
  - openjdk11
```

Reply:
this list looks better to me